dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -1188,29 +1098,47 @@ public Response fixConflicts ( @Context final HttpServletRequest httpServletRequ
         } catch (DotSecurityException e) {
             throw new ForbiddenException(e);
         } catch ( Exception e ) {
-        	try {
+            try {
                 HibernateUtil.rollbackTransaction();
             } catch (DotHibernateException e1) {
                 Logger.error(IntegrityResource.class, "Error while rolling back transaction", e);
             }
 
             Logger.error( this.getClass(), "Error fixing "+type+" conflicts for End Point server: [" + endpointId + "]", e );
             return response( "Error fixing conflicts for endpoint: " + endpointId , true );
-		} finally {
-			try {
-				// Discard conflicts if successful or failed
-				integrityUtil.discardConflicts(endpointId, integrityTypeToFix);
-			} catch (DotDataException e) {
-				Logger.error(this.getClass(), "ERROR: Table " + integrityTypeToFix.getResultsTableName()
-						+ " could not be cleared on end-point [" + endpointId
-						+ "]. Please truncate the table data manually.", e);
-			}
-			HibernateUtil.closeSessionSilently();
-		}
+        } catch (NotEndPointTokenFoundException e) {
+            return Response.status( HttpStatus.SC_BAD_REQUEST ).entity( "Error: 'auth key' is a required param." ).build();
+        } finally {
+            try {
+                // Discard conflicts if successful or failed
+                integrityUtil.discardConflicts(endpointId, integrityTypeToFix);
+            } catch (DotDataException e) {
+                Logger.error(this.getClass(), "ERROR: Table " + integrityTypeToFix.getResultsTableName()
+                        + " could not be cleared on end-point [" + endpointId
+                        + "]. Please truncate the table data manually.", e);
+            }
+            HibernateUtil.closeSessionSilently();
+        }
 
         return response( jsonResponse.toString(), false );
     }
 
+    @NotNull
```

Comment:
remove this

Reply:
done https://github.com/dotCMS/core/pull/19273/commits/303cd0bf044e9265865287ea596719a4346e2895#diff-1c1ba904ed03287c1c0b48a6f3971449L458